src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -202,51 +202,51 @@ addRemovedAccount addr@CWAddressMeta{..} = do
     wsWalletInfos . ix acc . wiRemovedAccounts . at addr ?= ()
 
 setAccountMeta :: AccountId -> CAccountMeta -> Update ()
-setAccountMeta cAddr wMeta = wsWalletInfos . ix cAddr . wiMeta .= wMeta
+setAccountMeta accId cAccMeta = wsWalletInfos . ix accId . wiMeta .= cAccMeta
 
 setWalletMeta :: CId Wal -> CWalletMeta -> Update ()
-setWalletMeta cAddr wSMeta = wsWSetInfos . ix cAddr . wsiMeta .= wSMeta
+setWalletMeta cWalId cWalMeta = wsWSetInfos . ix cWalId . wsiMeta .= cWalMeta
 
 setWalletPassLU :: CId Wal -> PassPhraseLU -> Update ()
-setWalletPassLU cAddr passLU = wsWSetInfos . ix cAddr . wsiPassphraseLU .= passLU
+setWalletPassLU cWalId passLU = wsWSetInfos . ix cWalId . wsiPassphraseLU .= passLU
 
 setWalletSyncTip :: CId Wal -> HeaderHash -> Update ()
-setWalletSyncTip cAddr hh = wsWSetInfos . ix cAddr . wsiSyncTip .= hh
+setWalletSyncTip cWalId hh = wsWSetInfos . ix cWalId . wsiSyncTip .= hh
 
 addAccountHistoryTx :: AccountId -> CTxId -> CTxMeta -> Update ()
-addAccountHistoryTx accId ctxId ctxMeta =
-    wsTxHistory . ix (aiWSId accId) . at ctxId ?= ctxMeta
```

Comment:
However I wished you to change functions like this one under this issue )
Despite it accepts `AccountId` and called `addAccountHistoryTx`, it actually gets history of whole wallet, so I propose to change argument type